draft-deepspace-ip-architecture.xml

```diff
@@ -48,31 +48,129 @@
-      <t>It should also be noted that DTN and BP were also designed for non-space use cases. While this document focuses on the deep space use case, it shall work for the other use cases of BP, but these use cases are out of scope of this document.</t>
-      <t>Space missions are typically planned many years in advance and are long-lived, spanning over many years even decades. Spacecrafts are controlled from Earth and therefore should always be manageable from Earth. Given the remoteness and the difficulty to physically access the spacecraft, software upgrades and configuration changes are avoided whenever possible.</t>
-      <t>As with Bundle protocol, this framework proposes to use IP in deep space with a similar store-and-forward paradigm. Therefore, the IP layer has to deal with the fact that a destination may not be currently reachable and that IP packets should be stored for an unusual amount of time, such as minutes or hours or days, in the forwarding device waiting for a new link up  opportunity. The transport layer should be able to work with long and variable delays, including intermittent communications. The application protocols and application themselves should be properly set to wait a longer time than on Internet to receive a response to a query. Finally, all network services such as routing, security, naming and network management should also be adapted in this new context. This document is structured around these layers.</t>
+      <t>
+	Deep space communications involve long delays (e.g., Earth to
+	Mars is 4-20 minutes) and intermittent communications, because
+	of orbital dynamics. Up to now, communications have been done on
+	a layer-2 point-to-point basis, with sometimes the use of
+	relays, but no layer-3 networking has been in use. <xref
+	target="RFC4838"/> reports an assessment done around 25 years
+	ago concluding that the IP protocol stack was not suitable for
+	deep space networking. This result led to the definition of a
+	completely new protocol stack based on a store-and-forward
+	paradigm implemented in the Bundle Protocol(BP) <xref
+	target="RFC9171"/> and its various components, such as
+	convergence-layer adapters (<xref target="RFC9174"/>, <xref
+	target="RFC7122"/>) and BP Security (BPSEC) <xref
+	target="RFC9172"/>.
+      </t>
+      <t>More recently, space agencies plan to deploy IP
```

Comment:
Suggest changing "space agencies" to something more like "several groups (including companies and national space agencies)".